python/composio/client/__init__.py

```diff
@@ -138,6 +139,21 @@ def http(self) -> HttpClient:
     @http.setter
     def http(self, value: HttpClient) -> None:
         self._http = value
+        
+    @property
+    def long_timeout_http(self) -> HttpClient:
+        if not self._long_timeout_http:
+            self._long_timeout_http = HttpClient(
+                base_url=self.base_url,
+                api_key=self.api_key,
+                runtime=self.runtime,
+                timeout=180.0,
```

Comment:
Consider making the timeout value configurable through a class constant or constructor parameter instead of hardcoding 180.0. This would make the timeout more flexible for different deployment scenarios.